src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComMethodContext.cs

```diff
@@ -105,6 +105,7 @@ private MethodDeclarationSyntax CreateUnreachableExceptionStub()
         {
             // DeclarationCopiedFromBaseDeclaration(<Arguments>) => throw new UnreachableException("This method should not be reached");
```

Comment:
I don't think we are emitting a string. 

```suggestion
            // DeclarationCopiedFromBaseDeclaration(<Arguments>) => throw new UnreachableException();
```